components/resources/quizzes/shared/BluePrint.tsx

```diff
@@ -40,7 +52,10 @@ const BluePrint = ({ onNextClick, onBackClick }: BluePrintProps) => {
       <div className="grid py-6 place-items-center">
         <Button
           label="View results"
-          onClick={onNextClick}
+          onClick={() => {
+            onNextClick();
+            handleQuizResponseMutations(quizViewState, result);
```

Comment:
can we just make this function part of the `onNextClick` function that's passed in from the parent?